components/separator.json

```diff
@@ -1,8 +1,12 @@
 {
   "separator": {
```

Comment:
There are 2 components affected by this new token.
1. platformcomponents/desktop/navigation-tab : this file already exists. We just need to add the divider line that separates the 1st column with the rest of the app. https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5250%3A957
```
{
    "navigationtab":{
        "comment": "Applied to the left nav tabs. The badge is a separate component.",
        "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4342%3A14155",
        "background":{...},
        "text":{...},
        **"divider":
               "normal":"@theme-gradientdivider-normal",
               "hover": ...
               ....**
    }
}
```
We would need to ask the design team what each block of nav tabs means on [that Figma](https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5250%3A957).

Reply:
2. The compose area, which is a new component that will be added to the library soon. Just add this one if you need it for your work. Otherwise, I can add it when they officially include it on the library. The Figma link for this component is here: https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=5415%3A0
The component would look sth like this:
```
"composearea":{
     "divider":{
        ...
     }
}
```
I am asking the design team if this is a desktop-only component, or whether mobile would also have it. In the first case, this component would need to be added to `platformcomponents/desktop` folder. In the second case, it would simply need to be added to `components` folder.

Reply:
These dividers are the same component for desktop though, so it would make sense for us to have the token definition done in one place. I can move this under `platformcomponents/desktop` if you think that'd make more sense?

Reply:
correct; they also confirmed me that it would only be for desktop